i2c/26-i2c.js

```diff
@@ -126,17 +127,19 @@ module.exports = function(RED) {
             if (isNaN(address)) address = msg.address;
             var command = node.command;
             if (isNaN(command)) command = msg.command;
+            var buffcount = node.count;
+            if (isNaN(buffcount)) buffcount = msg.bytes;
             address = parseInt(address);
             command = parseInt(command);
-            var buffcount = parseInt(node.count);
+            buffcount = parseInt(buffcount);
```

Comment:
again- need to check msg.bytes is a valid number

Reply:
also already done a few lines later (if statement at line 141)

Reply:
yes- ok